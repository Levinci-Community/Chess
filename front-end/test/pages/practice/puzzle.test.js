import React from "react";
import { render } from "@testing-library/react";
import PuzzlePage from "../../../src/pages/practice/puzzle/index";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders PuzzlePage snapshot", () => {
    const { container } = render(
      <Router>
        <PuzzlePage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });