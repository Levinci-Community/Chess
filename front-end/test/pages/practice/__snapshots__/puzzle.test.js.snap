// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PuzzlePage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-1nyhgc2"
  >
    <h2
      class="chakra-heading css-bn3r0t"
    >
      puzzles.puzzle
    </h2>
    <div
      class="css-k008qs"
    >
      <div
        class="css-1q77jyp"
      >
        <div
          class="css-9zdb0q"
        >
          <div
            class="chakra-aspect-ratio css-1ghdmhg"
          >
            <iframe
              allowfullscreen=""
              id="chess-puzzle"
              src="https://livetactics.chessbase.com"
              title="Chess Tactics"
            />
          </div>
          <div
            class="css-1dsnzft"
          />
        </div>
      </div>
      <div
        class="css-18467a"
      />
    </div>
  </div>
</div>
`;
